cms/static/js/views/edit_chapter.js

```diff
@@ -52,8 +52,10 @@ define(["js/views/baseview", "underscore", "underscore.string", "jquery", "gette
                 asset_path: this.$("input.chapter-asset-path").val()
             });
             var msg = new FileUploadModel({
-                title: _.template(gettext("Upload a new PDF to “<%= name %>”"),
-                    {name: course.escape('name')}),
+                title: _.template(
+                    gettext("Upload a new PDF to “<%= name %>”"),
+                    {name: window.course.escape('name')}
```

Comment:
I had to punt on getting rid of this escaping and moving it to the File Upload.  This PR had already grown out of hand.
See [this TNL ticket](https://openedx.atlassian.net/browse/TNL-4077?focusedCommentId=169512&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-169512) if you want to join a discussion around how we might handle this in the future.